src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1715,7 +1715,7 @@ def update_container_settings(cmd, resource_group_name, name, docker_registry_se
         except Exception as ex:  # pylint: disable=broad-except
             logger.warning("Retrieving credentials failed with an exception:'%s'", ex)  # consider throw if needed
 
-    if docker_registry_server_user is not None:
+    if docker_registry_server_user is not None: 
```

Comment:
I would remove this space since running `azdev style appservice` would throw an error for it